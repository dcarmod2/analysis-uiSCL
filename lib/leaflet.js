import Leaflet from 'leaflet'

/**
 * There were universal settings for Leaflet that made importing it from a local
 * file instead of the library a good idea.
 */
export default Leaflet
