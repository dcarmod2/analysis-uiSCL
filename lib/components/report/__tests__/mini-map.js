import enzyme from 'enzyme'
import {latLngBounds} from 'leaflet'
import React from 'react'

import MiniMap from '../mini-map'

describe('Report > MiniMap', () => {
  it('renders correctly', () => {
    const props = {
      bounds: latLngBounds([
        [40.712, -74.227],
        [40.774, -74.125]
      ])
    }

    const tree = enzyme.shallow(<MiniMap {...props} />)
    expect(tree).toMatchSnapshot()
  })
})
