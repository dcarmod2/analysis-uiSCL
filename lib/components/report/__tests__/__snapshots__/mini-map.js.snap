// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report > MiniMap renders correctly 1`] = `
<div
  style={
    Object {
      "height": 200,
      "width": 720,
    }
  }
>
  <Map
    attributionControl={false}
    bounds={
      Object {
        "_northEast": Object {
          "lat": 40.774,
          "lng": -74.125,
        },
        "_southWest": Object {
          "lat": 40.712,
          "lng": -74.227,
        },
      }
    }
    boxZoom={false}
    doubleClickZoom={false}
    dragging={false}
    keyboard={false}
    scrollWheelZoom={false}
    touchZoom={false}
    zoomControl={false}
  >
    <ForwardRef(Leaflet(MapBoxGLLayer)) />
  </Map>
</div>
`;
